kirin/piv/model_maker.py

```diff
@@ -304,14 +308,15 @@ def build_trip_updates(self, rt_update):
         # replace by cleaned and sorted version of stoptimes list.
         json_train["listeArretsDesserte"]["arret"] = ads
 
-        vj = self._get_navitia_vj(piv_key, ads)
+        trip_addition = higher_disruption.get("type") == "CREATION"
```

Comment:
Suggesting 2 remanes:
* As bool are mostly named like that,
* As I always have :cold_sweat: thinking disruption is the higher among stops
```suggestion
        is_trip_addition = higher_trip_disruption.get("type") == "CREATION"
```